cvat-canvas3d/README.md

```diff
@@ -0,0 +1,49 @@
+# Module CVAT-CANVAS-3D
+
+## Description
+
+The CVAT module written in TypeScript language.
+It presents a canvas to viewing, drawing and editing of 3D annotations.
+
+## Versioning
+
+If you make changes in this package, please do following:
+
+- After not important changes (typos, backward compatible bug fixes, refactoring) do: `npm version patch`
+- After changing API (backward compatible new features) do: `npm version minor`
+- After changing API (changes that break backward compatibility) do: `npm version major`
+
+## Commands
+
+- Building of the module from sources in the `dist` directory:
+
+```bash
+npm run build
+npm run build -- --mode=development     # without a minification
+```
+
+### API Methods
+
+```ts
+interface Canvas3d {
+  html(): HTMLDivElement;
+  setup(frameData: any, objectStates: any[], zLayer?: number): void;
+  fitCanvas(): void;
+  mode(): Mode;
+  configure(configuration: Configuration): void;
```

Comment:
The method is not presented on class ``Canvas3dImpl``

Reply:
Resolved